benchmarks/bench_plot_polynomial_kernel_approximation.py

```diff
@@ -30,12 +30,12 @@
 [1] Pham, N., & Pagh, R. (2013, August). Fast and scalable polynomial
 kernels via explicit feature maps. In Proceedings of the 19th ACM SIGKDD
 international conference on Knowledge discovery and data mining (pp. 239-247)
-(http://chbrown.github.io/kdd-2013-usb/kdd/p239.pdf)
+(https://chbrown.github.io/kdd-2013-usb/kdd/p239.pdf)
 
 [2] Charikar, M., Chen, K., & Farach-Colton, M. (2002, July). Finding frequent
 items in data streams. In International Colloquium on Automata, Languages, and
 Programming (pp. 693-703). Springer, Berlin, Heidelberg.
-(http://www.vldb.org/pvldb/1/1454225.pdf)
+(https://vldb.org/pvldb/vol1/1454225.pdf)
```

Comment:
This URL appears to be incorrect! It is pointing to a "Finding frequent items in data streams" paper by Cormode, G. and Hadjileftheriou, M. from 2008, **not** to the paper with the same title by Charikar, M., Chen, K., & Farach-Colton, M. from 2002. The latter would be:
* https://link.springer.com/chapter/10.1007/3-540-45465-9_59

Reply:
It looks like the PDF is from the initial conference paper from 2002. The algorithm steps are named _CandidateTop_ and _ApproxTop_ as in the conference paper from 2002 instead of  _FindCandidateTop_ and _FindApproxTop_ in the journal paper from 2004.

Therefore I will link to the latter PDF URL.